src/Main.js

```diff
@@ -1,19 +1,26 @@
-/*
- * To change this license header, choose License Headers in Project Properties.
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
- */
+// Modules using `default` export must be imported...
```

Comment:
Some exports are missing (GeometricLayer for example). I think a good test to see if enough classes are exposed is to try to recreate PlanarView or GlobeView from outside the core.